### PR TITLE
Added domaindiscount24

### PR DESCRIPTION
**Pull request by rugk**
#2324 

(previously PR https://github.com/EFForg/https-everywhere/pull/2487 - I had some problems with the branches)

### CODE REVIEW

File: src/chrome/content/rules/domaindiscount24.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-->
+<ruleset name="Domain Discount 24">
+	<target host="domaindiscount24.com" />
+	<target host="www.domaindiscount24.com" />
+	<target host="domaindiscount24.net" />
+	<target host="www.domaindiscount24.net" />
+	<target host="webmail.domaindiscount24.com" />
+	<target host="login.domaindiscount24.com" />
+
+  <securecookie host="^(www|webmail|login\.)?domaindiscount24\.com$" name=".+" />
```
Comment (julianladisch) [inline src/chrome/content/rules/domaindiscount24.xml]: Change

    (www|webmail|login\.)

to

    (www\.|webmail\.|login\.)

Comment (rugk) [inline src/chrome/content/rules/domaindiscount24.xml]: Oh, yes, thanks for noticing me.
Fixed with https://github.com/EFForg/https-everywhere/commit/f5b4cc05521053bec7c3a994983369ead303990c.

**Comment by rugk** [comment]
Does anything speaks against merging this?

**Comment by jsha** [comment]
Hi @rugk! I've been focusing on merging fixes over new rules, just as a simple way to conserve my time to work on Let's Encrypt. Sorry that some of your PRs are dragging on so long!

**Pull request merged**